CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Features
 - Added support for callbacks that stop the synchronization if the lag or lack of progress exceeds a certain threshold ([#1478](https://github.com/neptune-ai/neptune-client/pull/1478))
+- Add support for `retry-after` header in HTTPTooManyRequests ([#1477](https://github.com/neptune-ai/neptune-client/pull/1477))
```

Comment:
Features are mostly user-exposed. This is probably a Change. What do you think?

Reply:
Oh, that makes sense! Moved 😄